ReactSkia/components/RSkComponentTextInput.cpp

```diff
@@ -48,7 +50,8 @@ RSkComponentTextInput::RSkComponentTextInput(const ShadowView &shadowView)
   cursorPaint_.setAntiAlias(true);
   cursorPaint_.setStyle(SkPaint::kStroke_Style);
   cursorPaint_.setStrokeWidth(CURSOR_WIDTH);
-  
+  sem_init(&jsUpdateMutex,0,0);
```

Comment:
Can sem_destroy be added as a counter part in ~RSkComponentTextInput

Reply:
addressed